pkg/driver/node.go

```diff
@@ -494,16 +494,11 @@ func (ns *node) NodeUnpublishVolume(
 	}
 
 	notMnt, err := ns.mounter.IsLikelyNotMountPoint(target)
-	if err != nil && !os.IsNotExist(err) {
+	if (err == nil && notMnt) || os.IsNotExist(err) {
 		logrus.Warningf("NodeUnpublishVolume: %s is not mounted, err: %v", target, err)
 		return &csi.NodeUnpublishVolumeResponse{}, nil
 	}
```

Comment:
What if there is an error?

Reply:
if there is an error still we should try to unmount